Module/randomize.py

```diff
@@ -117,12 +146,22 @@ def setKeybladeAbilities(self, keybladeAbilities = ["Support"], keybladeMinStat
 
     def setRewards(self, levelChoice="ExcludeFrom50", betterJunk=False):
         locations = [location for location in self._validLocationList if not isinstance(location, KH2ItemStat)]
+        location_weights = [location.LocationWeight for location in locations]
         for item in self._validItemList:
+            weighted_random = item.Id in [593,594,595,21,22,23,24,87,88,26,27,29,31,
```

Comment:
I'm not sure if we currently have an list of important checks somewhere that is easy to import from, but if not we should move this to be a constant somewhere (or maybe in a function called something like "getImportantChecks"). 

This would better future proof us since having customizable important checks seems like an obvious feature to add someday.

Reply:
I agree about adding an important check getter function, but of note, this is not just important checks but "useful" checks like munny pouch, finishing plus, etc. I can make an equivalent `getUsefulChecks` as well to cover this.